src/components/dashboard/FaceRecognition/FaceRecognition.js

```diff
@@ -176,12 +138,13 @@ class FaceRecognition extends React.Component<FaceRecognitionProps, State> {
           />
         )}
 
-        {this.state.zoomReady && showZoomCapture && (
-          <ZoomCapture
-            screenProps={this.props.screenProps}
-            onCaptureResult={this.onCaptureResult}
-            showZoomCapture={this.state.zoomReady && showZoomCapture}
-            loadedZoom={this.loadedZoom}
+        {showCamera && (
+          <FaceCapture
+            pictureOptions={{
+              width: 1080,
+            }}
+            debug={true}
```

Comment:
debug should be true only in development

Reply:
removed this param

Reply:
@StanislavShevchenko this should be true if env===development

Reply:
@StanislavShevchenko create src/lib/utils/devMode.js helper:
```
import Config from '../../config/config';

export const devMode = 'development' == Config.env;

export const logMessage = (...args) => {
  if (!devMode) {
    return;
  }

  console.log(...args);
}
```

then just 
```
import { devMode } from '../relative/path/to/utils/devMode';
```

```
 debug={devMode}
```


Also pls seek where env === development check also is performing and adjust code found to use utils/devMode